### PR TITLE
Aaron dev message tests

### PR DESCRIPTION
**Pull request by AaronKutch**

### CODE REVIEW

File: x/market/keeper/msg_server_redeem_drop.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func (k msgServer) RedeemDrop(goCtx context.Context, msg *types.MsgRedeemDrop) (
 
 	poolProduct := member1.Balance.Mul(member2.Balance)
 
-	// Each Drop is a proportional rite to the AMM balances
+	// Each Drop is a proportional rate to the AMM balances
```
Comment (cgdusek) [inline x/market/keeper/msg_server_redeem_drop.go]: It is actually right, not rite or rate.  Each drop is a proportional entitlement / right to the AMM balances.  Just learned rite is a ritual and right is an entitlement.

File: x/market/types/message_create_order.go [hunk 2]
```diff
@@ -55,15 +55,28 @@ func (msg *MsgCreateOrder) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "invalid order type")
 	}
 
+	denomAsk, _ := sdk.ParseCoinNormalized(msg.DenomAsk)
```
Comment (cgdusek) [inline x/market/types/message_create_order.go]: These are just the denom names (nom), not coinStr (20nom).  ParseCoinNormalize needs coinStr.

File: x/market/types/message_create_order_test.go [hunk 3]
```diff
@@ -20,9 +20,269 @@ func TestMsgCreateOrder_ValidateBasic(t *testing.T) {
 				Creator: "invalid_address",
 			},
 			err: sdkerrors.ErrInvalidAddress,
-		}, {
+		},
+		{
 			name: "valid address",
-			msg:  MsgCreateOrder{Creator: sample.AccAddress(), DenomAsk: "20CoinA", DenomBid: "20CoinB", Rate: []string{"10", "20"}, OrderType: "stop", Amount: "10", Prev: "0", Next: "0"},
+			msg: MsgCreateOrder{
```
Comment (cgdusek) [inline x/market/types/message_create_order_test.go:26]: All of these have coinStr for DenomAsk and DenomBid, but they aren't being used as coinStr in the actual code.  The reason why the test passes, but a coinStr could be a Denom Name as well.  For instance 20CoinA could be a denom name and its associated coinStr could be 20CoinA20

File: x/market/keeper/msg_server_redeem_drop.go (hunk 1 above)
Comment (AaronKutch) [reply x/market/keeper/msg_server_redeem_drop.go]: TIL

File: x/market/types/message_create_order.go (hunk 2 above)
Comment (AaronKutch) [reply x/market/types/message_create_order.go]: Should I use `ValidateDenom` instead?

Comment (cgdusek) [reply x/market/types/message_create_order.go]: Yes

**Pull request merged**